keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/config/ClouddriverConfiguration.kt

```diff
@@ -45,13 +45,13 @@ class ClouddriverConfiguration {
   fun clouddriverService(
     clouddriverEndpoint: HttpUrl,
     objectMapper: ObjectMapper,
-    retrofitClient: OkHttpClient
+    clientProvider: OkHttpClientProvider
```

Comment:
I don't see where this is being instantiated. Was that introduced in a previous PR?

Reply:
Yes. Here is the PR in kork.
https://github.com/spinnaker/kork/pull/639